test/uk/gov/hmrc/tai/connectors/EmploymentsConnectorSpec.scala

```diff
@@ -333,6 +423,80 @@ class EmploymentsConnectorSpec extends PlaySpec
  |          }]}
         }""".stripMargin
 
+
+  private val zeroCeasedEmployments =
```

Comment:
is this the same as zeroEmployments?

Reply:
slightly different json

Reply:
i have moved them nearer to make it easier to compare